src/renderer/components/output.tsx

```diff
@@ -85,13 +85,15 @@ export class Output extends React.Component<CommandsProps> {
     const lines = entry.text.split(/\r?\n/);
     const style: React.CSSProperties = entry.isNotPre
       ? { whiteSpace: 'initial' }
-      : {};
+      : { whiteSpace: 'pre' };
```

Comment:
@issacgerges One comment: what does this line do? I tried manually switching the `initial` and `pre` values and didn't actually get any changes on my end.

![screencast 2020-11-06 14-07-11](https://user-images.githubusercontent.com/16010076/98419395-d729d600-2039-11eb-99b7-7563b52be6d1.gif)

Reply:
heya @erickzhao, `white-space: pre` tells the browser that line-breaking and whitespace are already "correct" in the output. With normal whitespace rules, the browser will break lines to fill the (layout) box and collapse repeated whitespace (`<span>     Hello</span>` would become `Hello`). With this rule it will only break lines when there are explicit linebreaks in the output and will preserve all whitespace.

In your example reducing the viewport size should show the text wrapping and not wrapping depending on that setting. Really glad you mentioned this though, helped me realized there was some CSS I had missed and could cleanup in this PR. Will send an update in a sec